services/QuillLMS/client/app/bundles/Staff/components/comprehension/rulesAnalysis/ruleAnalysis.tsx

```diff
@@ -184,11 +184,16 @@ const RuleAnalysis = ({ history, match }) => {
     { name: "", attribute:"value", width: "750px" }
   ];
 
+  const extractHighlight = (highlightObject) => {
+    if (!highlightObject || !highlightObject.length || highlightObject.length < 1 || !highlightObject[0].text) return '';
```

Comment:
0 is falsy in Javascript, so you don't need the `highlightObject.length < 1` clause here.

Reply:
addressed